apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/TracingContext.java

```diff
@@ -189,6 +189,7 @@ public void extract(ContextCarrier carrier) {
 
         carrier.extractExtensionTo(span);
         this.correlationContext.extract(carrier);
+        this.correlationContext.handle(span);
```

Comment:
This should be changed to `carrier.extractCorrelationTo(span);` right? `handle` should be hidden.